tasks/install.yml

```diff
@@ -11,6 +11,11 @@
     - "install/{{ ansible_distribution }}.yml"
     - "install/{{ ansible_os_family }}.yml"
 
+- name: Check if the systemd unit is installed by the package
+  stat:
+    path: /lib/systemd/system/haproxy.service
+  register: haproxySvcUnit
```

Comment:
We are facing a consistency issue. This is a camelCase typography and with Ansible we are used to use snake_case typography.

Could you please be so kind to convert this register variable?